app/views/components/docs/share-links.yml

```diff
@@ -0,0 +1,23 @@
+name: Share links
+description: Social media links, linking to facebook and twitter share pages
```

Comment:
Suggestion for alternative description: links to allow the user to share the current page on Facebook or Twitter